data/releases/schedule.yaml

```diff
@@ -1,123 +1,128 @@
----
-# This file helps to populate the /releases page, and is also parsed to find out the
-# latest patch version for a minor release.
 schedules:
-- release: 1.30
-  releaseDate: 2024-04-17
+- endOfLifeDate: "2025-06-28"
```

Comment:
This is missing 1.30.0 as a previous release

Reply:
(Given that https://github.com/kubernetes/website/pull/45018 has merged) I don't think we need to list v1.30.0 as a previous patch until the v1.30.1 patch release happens.

Reply:
I agree, the first patch is 1.30.1 technically.

Reply:
This is not really about skew, a lot of folks complained about usability because you couldn't get all the information on a single web page (i.e. the patch releases page). I'm +1 to keep it regardless of the fix (but it's nice that this is fixed!)

Reply:
See https://github.com/kubernetes/website/pull/39086 and https://kubernetes.slack.com/archives/C2C40FMNF/p1674214109801969 for the original context of this change

Reply:
Hm, I'll add it for now for sake of consistency.